vignettes/custom_appearance.Rmd

```diff
@@ -90,3 +94,239 @@ lyt3 <- basic_table() %>%
 tbl3 <- build_table(lyt3, ex_adsl)
 tbl3
 ```
+
+### Top-left Materials
+
+The sequence of strings printed in the area between the column header display and the first row label \emph{top left material} can be modified during pre-processing within the table constructor `basic_table()` using `append_topleft`, the label position in row splits `split_rows_by` and during post-processing using the `top_left()` function. Note: No structure or indenting is applied to these strings.
+
+Within the table construction:
+
+```{r}
+lyt <- basic_table() %>%
+  split_cols_by("ARM") %>%
+  split_rows_by("STRATA1") %>%
+  analyze("AGE") %>%
+  append_topleft("New top_left material here")
+
+build_table(lyt, DM)
+```
+
+Specify label position using the `split_rows` function. Notice the position of `STRATA1` and `SEX`.
+
+```{r}
+lyt <- basic_table() %>%
+  split_cols_by("ARM") %>%
+  split_rows_by("STRATA1", label_pos = "topleft") %>%
+  split_rows_by("SEX", label_pos = "topleft") %>%
+  analyze("AGE")
+
+build_table(lyt, DM)
+```
+
+Post-processing using the `top_left()` function:
+
+```{r}
+lyt <- basic_table() %>%
+  split_cols_by("ARM") %>%
+  split_rows_by("SEX") %>%
+  analyze(c("AGE", "STRATA1"), function(x) {
+    if (is.numeric(x)) {
+      in_rows(
+        "mean" = rcell(mean(x)),
+        "sd" = rcell(sd(x)),
+        .formats = c("xx.x"), .aligns = "left"
+      )
+    } else if (is.factor(x)) {
+      rcell(length(unique(x)), align = "right")
+    } else {
+      stop("Unsupported type")
+    }
+  }, show_labels = "visible", na_str = "NE") %>%
+  build_table(ex_adsl)
+
+# Adding top-left material
+top_left(lyt) <- "New top-left material here"
+
+lyt
+```
+
+### Table Inset
+
+Table title, table body, referential footnotes and and main footers can be inset from the left alignment of the titles and provenance footer materials. This can be modified within the table constructor `basic_table()` or during post-processing with `table_inset()`.
+
+Using the table constructor:
```

Comment:
subtle technical point: `basic_table` is not a table constructor (`TableTree()` and `ElementaryTable()` are). `basic_table` is actually a layout constructor (arguably it should be called `basic_layout` but that ship sailed long ago.

Even more relevant, though is that the purpose of `basic_table` is *layout initialization* because it constructs a layout that doesn't declare any table structure, and the user is expected to modify it to declare the table they want. Please call it the "layout initializer" if you do not want to just call it directly by name.